data/crac/crac-result-extensions/src/main/java/com/farao_community/farao/data/crac_result_extensions/ResultVariantManager.java

```diff
@@ -170,22 +170,23 @@ public void deleteVariants(String... variantIds) {
         }
     }
 
-    /**
-     * Computes a string that is not present in the set, creates a new variant from that string, and returns the string.
-     */
-    public String createNewUniqueVariantId() {
-        return createNewUniqueVariantId("");
-    }
-
     /**
      * Computes a string with a prefix that is not present in the set, creates a new variant from that string, and returns the string.
      */
-    public String createNewUniqueVariantId(String prefix) {
+    public String getUniqueVariantId() {
         String s = "";
         do {
-            s = prefix.concat(UUID.randomUUID().toString());
+            s = s.concat(UUID.randomUUID().toString());
         } while (variants.contains(s));
-        createVariant(s);
         return s;
```

Comment:
The idea of having a prefix was to be able to tell easily if it was a preOptim variant or a postOptim variant.

(if you really want to drop the prefix idea, you can also remove the concatenation and have 
s = UUID.randomUUID().toString()
)

Reply:
Would it be useful to keep it ? Maybe for debug purpose ? anything else ?

Reply:
it's mainly for debug yeah (and when we export the final json it's easier to distinguish which values are the inital ones and which are after the RAO)

Reply:
I agree with @phiedw, I think its nice to keep those prefix so that the CracResults can be "humanly understandable"